### PR TITLE
[SDPA-3735] Added youtube to social media list.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3735

### Changed

1.  Added youtube to the socila media list paragraph field.

### CODE REVIEW

File: tide_landing_page.install [hunk 1]
```diff
@@ -517,3 +518,17 @@ function tide_landing_page_update_8014() {
...
 }
+
+/**
+ * Adds youtube option paragraph type to field_paragraph_social_list.
+ */
+function tide_landing_page_update_8015() {
+  $field_storage = FieldStorageConfig::loadByName('paragraph', 'field_paragraph_social_list');
+  $new_field_storage = $field_storage->toArray();
+  $new_field_storage['settings']['allowed_values']['youtube_channel'] = 'Youtube channel';
+  $new_field_storage = FieldStorageConfig::create($new_field_storage);
```
Comment (vincent-gao) [inline tide_landing_page.install]: why creating a new field config?

Comment (vincent-gao) [reply tide_landing_page.install]: @MdNadimHossain , you might want to do $field_storage->setSettings(......)->save();

Comment (MdNadimHossain) [reply tide_landing_page.install]: Hi @vincent-gao 
Just saving it didn't work when updating the storage config, so followed the instructions from here.
https://www.drupal.org/docs/8/api/update-api/updating-entities-and-fields-in-drupal-8#s-updating-field-storage-config-items

Comment (vincent-gao) [reply tide_landing_page.install]: Hi @MdNadimHossain ,the instruction is for changing field type.but for changing the value.
if you do push your `youtube_channel` to `allowed_values`. and `$field_storage-> setSetting('allowed_values', $new_field_storage['settings']['allowed_values'])->save()` it should work. I think

Comment (MdNadimHossain) [reply tide_landing_page.install]: Ok, I will try to build locally like this again, if it works I will update this. thx

Comment (MdNadimHossain) [reply tide_landing_page.install]: @vincent-gao  I have made the changes. I was converting it to toArray() first, that is why that way wasn't working. But now directly did it without converting it.

**Pull request merged**